packages/core/src/modules/credentials/formats/jsonld/JsonLdCredentialFormat.ts

```diff
@@ -1,25 +1,48 @@
+import type { JsonObject } from '../../../../types'
+import type { SingleOrArray } from '../../../../utils'
+import type { IssuerOptions } from '../../../vc/models/credential/Issuer'
 import type { W3cCredential } from '../../../vc/models/credential/W3cCredential'
 import type { CredentialFormat } from '../CredentialFormat'
 import type { JsonLdOptionsRFC0593 } from './JsonLdOptionsRFC0593'
 
-export interface JsonLdAcceptRequestOptions extends SignCredentialOptionsRFC0593 {
-  verificationMethod?: string
+export interface JsonCredential {
+  '@context': Array<string> | JsonObject
+  id?: string
+  type: Array<string>
+  issuer: string | IssuerOptions
+  issuanceDate: string
+  expirationDate?: string
+  credentialSubject: SingleOrArray<JsonObject>
 }
-
-export interface SignCredentialOptionsRFC0593 {
-  credential: W3cCredential
+export interface JsonLdSignCredentialFormat {
+  credential: JsonCredential
   options: JsonLdOptionsRFC0593
 }
 
+// use empty object in the acceptXXX jsonld format interface so we indicate that
+// the jsonld format service needs to be invoked
+type EmptyObject = Record<string, never>
+
+// it is an option to provide the verification method in acceptRequest
+export interface JsonLdCreateRequestFormat {
+  verificationMethod?: string
+}
+
 export interface JsonLdCredentialFormat extends CredentialFormat {
   formatKey: 'jsonld'
   credentialRecordType: 'w3c'
   credentialFormats: {
-    createProposal: SignCredentialOptionsRFC0593
-    acceptProposal: SignCredentialOptionsRFC0593
-    createOffer: SignCredentialOptionsRFC0593
-    acceptOffer: SignCredentialOptionsRFC0593
-    createRequest: SignCredentialOptionsRFC0593
-    acceptRequest: JsonLdAcceptRequestOptions
+    createProposal: JsonLdSignCredentialFormat
+    acceptProposal: EmptyObject
+    createOffer: JsonLdSignCredentialFormat
+    acceptOffer: EmptyObject
+    createRequest: JsonLdSignCredentialFormat
+    acceptRequest: JsonLdCreateRequestFormat
+  }
+  formatData: {
+    proposal: JsonLdSignCredentialFormat
+    offer: JsonLdSignCredentialFormat
+    request: JsonLdSignCredentialFormat
+    credential: W3cCredential
```

Comment:
This will be a JSON representation of the credential, not an instance of W3cCredential.

I think we should create a new `JsonVerifiableCredential`  that extends `JsonCredential` and adds the proof property:

```
export interface JsonVerifiableCredential extends JsonCredential {
  proof: SingleOrArray<{
    type: string
    proofPurpose: string
    verificationMethod: string
    created: string
	  domain?: string
	  challenge?: string
	  jws?: string
	  proofValue?: string
	  nonce?: string
[key: string]: unknown
  }>
}
```

Reply:
Took out LinkedDataProof, replaced with above interface